### PR TITLE
add zh /docs/reference/glossary/cluster-operations.md

### PR DESCRIPTION
**Pull request by lichuqiang**
to address https://github.com/k8smeetup/website-tasks/issues/3141

### CODE REVIEW

File: content/zh/docs/reference/glossary/cluster-operations.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Cluster Operations
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/cluster-operations.md]: pls translate the titlee

File: content/zh/docs/reference/glossary/cluster-operations.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+id: cluster-operations
+date: 2019-05-12
+full_link:
+short_description: >
+ Activities such as upgrading the clusters, implementing security, storage, ingress, networking, logging and monitoring, and other operations involved in managing a Kubernetes cluster.
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/cluster-operations.md]: pls translate the short description.

File: content/zh/docs/reference/glossary/cluster-operations.md (hunk 1 above)
Comment (lichuqiang) [reply content/zh/docs/reference/glossary/cluster-operations.md]: done

File: content/zh/docs/reference/glossary/cluster-operations.md (hunk 2 above)
Comment (lichuqiang) [reply content/zh/docs/reference/glossary/cluster-operations.md]: done

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**